docs/user/README.md

```diff
@@ -269,31 +269,67 @@ variables is provided here. If an environment variable is not set, the specified
+[Docker Desktop](https://docs.docker.com/desktop/) or by using the corresponding commands:
+
+```console
+docker-compose -p modelardb-single start
+docker-compose -p modelardb-single stop
 ```
 
-After the [MinIO](https://min.io/) service is created, a [MinIO](https://min.io/) client is used to initialize
-the development bucket `modelardata`, if it does not already exist. [MinIO](https://min.io/) can be administered through
-its [web interface](http://127.0.0.1:9001). The default username and password, `minioadmin`, can be used to log in.
-A separate compose file is used for [MinIO](https://min.io/) so an existing [MinIO](https://min.io/) instance can be
-used when `modelardbd` is deployed using [Docker](https://docker.com/), if necessary.
+The single edge is running locally on port `9999` and can be accessed using the `modelardb` client or through 
+Apache Arrow Flight as described above. Tables can be created and data can be ingested, compressed, and saved to local 
+disk. The compressed data on local disk can then be queried.
 
-Similarly, the `modelardbd` instance can be built and started using the command:
+### Cluster deployment
+Once [Docker](https://docs.docker.com/) is set up, the cluster deployment can be started by running the following
+command from the root of the ModelarDB-RS repository:
 
 ```shell
-docker-compose -p modelardbd up
+docker-compose -p modelardb-cluster -f docker-compose-cluster.yml up
+```
+
+Note that `-p modelardb-cluster` is only used to name the project to make it easier to manage in
+[Docker Desktop](https://docs.docker.com/desktop/). Once created, the containers can be started and stopped using
+[Docker Desktop](https://docs.docker.com/desktop/) or by using the corresponding commands:
+
+```console
+docker-compose -p modelardb-cluster start
+docker-compose -p modelardb-cluster stop
 ```
 
-The instance can then be accessed using the Apache Arrow Flight interface at `grpc://127.0.0.1:9999`.
+The cluster deployment sets up a [MinIO](https://min.io/) object store and a [MinIO](https://min.io/) client is used to 
+initialize the bucket `modelardb` in the object store. [MinIO](https://min.io/) can be administered through its 
+[web interface](http://127.0.0.1:9001). The default username and password, `minioadmin`, can be used to log in. The 
+deployment also sets up a PostgreSQL database named `metadata` that is used as the metadata database for the cluster. 
+The username `modelardb_user` and the password `modelardb_password` can be used to access the database. 
+
+The cluster itself consists of a manager node, an edge node, and a cloud node. The manager node can be accessed using 
+the URL `grpc://127.0.0.1:9998`, the edge node using the URL `grpc://127.0.0.1:9999`, and the cloud node using the URL
+`grpc://127.0.0.1:9997`. Tables can be created through the manager node and data can be ingested, compressed, and 
```

Comment:
Maybe delete the text after `grpc://127.0.0.1:9997` as it seems to repeat what has been written above. If this is done, maybe combine the resulting short paragraph with the paragraph right above it about the cluster deployment setup. If the text is kept and the text above about querying cloud nodes is updated, this text should also be updated for consistency.

Reply:
I have kept it as it was since I think it makes sense to both introduce both deployments in the introduction of the section and also reiterate in the subsections what is possible now that the deployment is running.